include/deal.II/fe/fe_base.h

```diff
@@ -391,6 +398,14 @@ class FiniteElementData
   unsigned int
   n_dofs_per_cell() const;
 
+  /**
+   * Number of non-local dofs per cell. Non-local dofs means the degrees of
```

Comment:
it should be possible to do 

```
@copydoc non_local_dofs_per_cell
```

Please enable doxygen locally to see if builds correctly.